libfwupdplugin/meson.build

```diff
@@ -236,6 +236,20 @@ library_deps = [
   platform_deps,
 ]
 
+if have_rust
+  fwupdpluginrs = static_library(
+    'fwupdpluginrs',
+    sources: [
+      'fu-crc.rs',
+    ],
```

Comment:
is it not possible to link the rusty object files directly to fwupdplugin instead of a new static library?

Reply:
I don't *think* so; libfwupdplugin is `library()` which can't have type `staticlib`, and if you leave it out you get `ERROR: Rust target fwupdplugin contains a non-rust source file.`

Reply:
OK